src/packages/sdk/package.json

```diff
@@ -18,7 +18,7 @@
     "format": "prettier --write .",
     "lint": "eslint --cache --fix --ext .ts .",
     "lint-ci": "eslint --ext .ts .",
-    "test": "node ./scripts/patchJest && jest",
+    "test": "jest --forceExit --verbose --clearCache",
```

Comment:
--verbose is nice!
--forceExit was not needed before, do you know why it's needed now?
--clearCache that's new? why do we need this?

Reply:
removed --clearCache

Reply:
I have no idea why `--forceExit` is now needed. We would need to investigate.